auto-dark.el

```diff
@@ -178,22 +178,29 @@ already set the theme for the current dark mode state."
     (unless (eq appearance auto-dark--last-dark-mode-state)
       (auto-dark--set-theme appearance))))
 
+(defun auto-dark--update-frame-backgrounds (appearance)
+  "Sets the ‘frame-background-mode’ to all frames to APPEARANCE."
```

Comment:
I think the quote style for docs is a bit different:

‘frame-background-mode’ -> `frame-background-mode'

Reply:
Aww, I was going to link the section of the docs that recommends the style I used, but “Some previous versions of this section recommended using the non-ASCII single quotation marks directly in doc strings, but this is now discouraged, since that leads to broken help string displays on terminals that don’t support displaying those characters.” (from [the Elisp Manual, D.6](https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html#index-curved-quotes-1)).

It’s too bad, as I’ve always found the `` `' `` pairing to be the worst sort of typography (despite being an Emacs user for 26 years) and was happy for the reprieve.

But I’ll switch to ASCII. Thanks for the heads-up.